orbit-db-cli/.github/workflows/failureNotifications.yml

```diff
@@ -0,0 +1,43 @@
+name: failureNotifications
```

Comment:
I don't think we need this "slack failure notification" action too

Reply:
same